docs/query-language.md

```diff
@@ -925,3 +926,12 @@ GroupBy(Rows(blah, previous=39), Rows(blahk, previous="haha"), limit=7)
 [{"group":[{"field":"blah","rowID":39},{"field":"blahk","rowKey":"zaaa"}],"count":1},
  {"group":[{"field":"blah","rowID":39},{"field":"blahk","rowKey":"traa"}],"count":1}]
 ```
+
+Using the filter argument.
+```request
+GroupBy(Rows(blah, previous=39), Rows(blahk, previous="haha"), limit=7, filter=Row(age=22))
+```
+
+```response
+TODO
```

Comment:
we should do this... and add a realistic example

Reply:
Ah, I knew there was some reason I hadn't PRed this yet